.github/workflows/ci.yml

```
@@ -31,7 +31,7 @@ env:
 
```

Rather than repeatedly change the platform in 12 places how do you feel about using an environment variable?

```suggestion
  PLATFORM: ubuntu-22.-4
```

---

Oddly this does not seem possible? The documentation on contexts makes it seems that maybe this could be possible:

https://docs.github.com/en/actions/learn-github-actions/contexts

But I couldn't get it to work (see #5986 for one example attempt). Searching Google's internal code base using "f:yml runs-on.*\$", it seems like the only way others have gotten this to work is to use the `matrix` context but we already use `matrix` where appropriate.

---

That's a shame, thanks for looking into it. There is some merit to declaring the platform for each step anyway so I don't mind the current approach too much.